tests/discover/libraries/data/apache.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+# vim: dict+=/usr/share/beakerlib/dictionary.vim cpt=.,w,b,u,t,i,k
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+rlJournalStart
+    rlPhaseStartSetup
+        rlAssertRpm "httpd"
+        #rlRun "rlImport httpd/http"
```

Comment:
If we can't use rlImport yet, could you mark it with TODO or FIXME? 
While it is great that `source ../../` works _the_ supported way is rlImport and IMHO we should test that

Reply:
Makes sense added.